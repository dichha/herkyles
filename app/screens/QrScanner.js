import React, { Component } from "react"; 
import { View, Text, AppRegistry, StyleSheet, TouchableOpacity, Linking, StatusBar} from "react-native"; 
import { StackNavigator } from "react-navigation"; 
 
import Container from "../components/Container/Container";

class QR extends Component {
    static navigationOptions = {
        title: "Home", 
        headerStyle: {
            backgroundColor: "#000000",
        }, 
          headerTintColor: '#facf33',
    };

    render(){
        return (
            <Container>
                <StatusBar transclucent={false} barStyle="light-content"/>
                <Text style={{textAlign: 'center', fontSize: 15}}>QR Scanner</Text>
            </Container>
        );
    }
};

export default QR;