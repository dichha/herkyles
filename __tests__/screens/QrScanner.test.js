import React from "react";
import QrScanner from "./../app/screens/QrScanner";

import renderer from "react-test-renderer";

describe ("QR Scanner", () => {
  it("renders without crashing", () => {
    const rendered = renderer.create(<QrScanner />).toJSON();
    expect(rendered).toBeTruthy();
    expect(rendered).toMatchSnapshot();
  });
});