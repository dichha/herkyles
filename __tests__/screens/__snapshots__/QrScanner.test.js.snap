// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Qr Scanner renders without crashing 1`] = `<View />`;
